src/Components/Blazor/Blazor/src/Rendering/WebAssemblyRenderer.cs

```diff
@@ -120,7 +120,7 @@ protected override void HandleException(Exception exception)
         }
 
         /// <inheritdoc />
-        public override Task DispatchEventAsync(ulong eventHandlerId, EventFieldInfo eventFieldInfo, UIEventArgs eventArgs)
```

Comment:
I feel really strongly positive about this change this in general. The idea that `EventArgs` represents all events, and the renderer could dispatch *any* event feels like we got the concept right.